### DISCUSSION TITLE
Certification necessity

### DISCUSSION THREAD

**Discussion by strv** [openthread/openthread#10246] [Q&A] [answered]
Hi all

I'm a DIY gadget builder and selling a small number of them.
I'm considering making a new gadget that uses Open thread, but I am confused about certifications. The current design uses ESP32 H2 for the wireless device and uses its API. Do I need independent certification for my product? Or is it not required because ESP32 H2 is certified?

**Comment by chshu** [comment]
@strv Please refer to [Thread Certification](https://threadgroup.org/What-is-Thread/Certification) for more details.

Generally, ESP32-H2 is a certified component, and you need to apply for `Built On Thread` product certification. The `Certification by Inheritance` method may simplify the process.

**Reply by strv** [reply]
Thank you for your answer. Your clear answer is also helpful for me.

**Comment by jwhui** [comment, marked as answer]
Becoming a Thread Certified Component or Built on Thread certified product confers three benefits to those who successfully complete certification:
1. Rights to use word and logo marks.
1. Rights to claim compatibility.
1. Rights to intellectual property held by those organizations contributing to the Thread specification.

By not certifying, your organization is opting out of the benefit of all three of those and in particular, taking on the risk potential associated with (3). As always, the consultation of outside or inside counsel is advised to best assess the risks to your business.

If you are using an existing Thread Certified Component, you may qualify for the inheritance certification program available to Associate, Implementer, and Contributor members. The inherited certification and an Associate membership is the lowest cost way to enjoy the three benefits above.

Thread Group also has an [Innovation Enabler Program](https://www.threadgroup.org/Innovation-Enabler-Award), which helps emerging companies launch Thread-enabled products by granting them a 2-year complimentary Thread Group membership.

**Reply by strv** [reply]
Thank you for the very detailed and clear answer. I completely agree with your suggested benefits. Unfortunately, my product is a side business, and the sales are less than $1000 annually. Thus, attending as a member, even as an Implementer or Contributor is hard.
Thank you for notifying me of the Innovation Enabler Program. I'm not thinking of enlarging my side business but selling items in the long term, so the two-year program is not enough for me. I'll try to use raw 802.15.4.
